src/c++/perf_analyzer/genai-pa/genai_pa/parser.py

```diff
@@ -134,7 +183,8 @@ def parse_args(argv=None):
     add_model_args(model_group)
 
     profile_group = parser.add_argument_group("Profiling")
```

Comment:
Can you move these group variable creations inside the respective functions they're used. Doesn't seem like they're used later in this function.